ulib/axlibc/src/unistd.rs

```diff
@@ -27,3 +35,68 @@ pub unsafe extern "C" fn abort() -> ! {
 pub unsafe extern "C" fn exit(exit_code: c_int) -> ! {
     sys_exit(exit_code)
 }
+
+/// Sleep for a number of seconds.
+#[cfg(feature = "signal")]
+#[no_mangle]
+pub unsafe extern "C" fn alarm(seconds: c_uint) -> c_uint {
+    let it = ctypes::itimerval {
+        it_interval: ctypes::timeval {
+            tv_sec: 0,
+            tv_usec: 0,
+        },
+        it_value: ctypes::timeval {
+            tv_sec: seconds as i64,
+            tv_usec: 0,
+        },
+    };
+    let mut old = ctypes::itimerval {
+        it_interval: ctypes::timeval {
+            tv_sec: 0,
+            tv_usec: 0,
+        },
+        it_value: ctypes::timeval {
+            tv_sec: 0,
+            tv_usec: 0,
+        },
+    };
+    if getitimer(ctypes::ITIMER_REAL as i32, &mut old) < 0 {
+        e(sys_setitimer(ctypes::ITIMER_REAL as i32, &it));
+        1
```

Comment:
return 1 or return e(sys_setitimer(ctypes::ITIMER_REAL as i32, &it))?

Reply:
return e(sys_setitimer(ctypes::ITIMER_REAL as i32, &it)) is right